src/content/docs/accounts/accounts-billing/new-relic-one-user-management/authentication-domains-saml-sso-scim-more.mdx

```diff
@@ -88,18 +88,57 @@ Notes on these settings:
 
 For how to set up SCIM, see [Automated user management](/docs/accounts/accounts/automated-user-management/automated-user-provisioning-single-sign).
 
-## Authentication: username/password versus SAML SSO  [#authentication]
+### User type upgrade requests [#user-upgrade]
+
+In the **Source of users** UI, you have two options for how your users' [user type](/docs/accounts/accounts-billing/new-relic-one-user-management/user-type) is managed:
+
+* **Manage user type in New Relic**: This is the default option. It allows you to manage your users' user type from New Relic.
+* **Manage user type with SCIM**: Enabling this means that **you can no longer manage user type from New Relic**. You'd only be able to change and manage it from your identity provider.
+
+More on these two options:
+
+<CollapserGroup>
+  <Collapser
+    id="manage-user-type-new-relic"
+    title="Manage user type in New Relic (default)"
+  >
+    The default way to manage your users' user type is from the New Relic UI. This is the option you'd use if you aren't [managing user type via SCIM](#manage-user-type-scim). The management options governed by this include:
```

Comment:
What do you think of "manage a user type"? I understand why we have users' user type but I'm just wondering if there's a way to remove the repetition

Reply:
i've thought about that damn thing a lot and unfortunately i think it's better than a lot of other options I brainstormed. i agree it's a bit awkward but the alt ways are also awkward